recipes/dunovo/meta.yaml

```diff
@@ -0,0 +1,41 @@
+
+package:
+  name: dunovo
+  version: 0.6
+
+source:
+  fn: v0.6.tar.gz
+  url: https://github.com/galaxyproject/dunovo/archive/v0.6.tar.gz
+  sha256: a3cc378c22fcfc3bcaa11129f27f61bab5ff51cc9185320d53ece273b56e6214
+
+build:
+  number: 0
+
+requirements:
+  skip: True  # [py3k or osx]
+  build:
+    - make
+    - gcc
+
+  run:
+    - libgcc
+    - python >=2.6,<3
+    - argparse  # [py26]
```

Comment:
Thanks for taking a look! I saw this trick in another package. Argparse was added to the standard library in 2.7, so if Python is 2.7, no dependency needs to be installed. For Python 2.6, it's not in the standard library, but it can be installed from PyPI, hence the dependency.

As for python<3, I just figured it's nice to be safe and explicit.